module/VuFind/src/VuFind/Controller/CombinedController.php

```diff
@@ -339,21 +339,32 @@ protected function adjustQueryForSettings($settings, $searchType = null)
         // Override the search type:
         $query->type = $searchType;
 
-        // Always leave noresults active (useful for 0-hit searches).
-        // Display or hide top based on include_recommendations setting.
-        // Display or hide side based on include_recommendations_side setting.
         $recommendOverride = [];
         $noRecommend = [];
+
+        // Display or hide top based on include_recommendations setting.
         if (is_array($settings['include_recommendations'] ?? false)) {
             $recommendOverride['top'] = $settings['include_recommendations'];
         } else {
             $noRecommend[] = 'top';
         }
+
+        // Display or hide side based on include_recommendations_side setting.
         if (is_array($settings['include_recommendations_side'] ?? false)) {
             $recommendOverride['side'] = $settings['include_recommendations_side'];
         } else {
             $noRecommend[] = 'side';
```

Comment:
I am not adding anything to $noRecommend[] for noresults or noresults_side, because (per the comment above) I think I am missing something about how they used to be active.